src/elements/emby-playstatebutton/PlayedButton.tsx

```diff
@@ -0,0 +1,74 @@
+import { BaseItemKind } from '@jellyfin/sdk/lib/generated-client';
+import React, { FC, useCallback } from 'react';
+import CheckIcon from '@mui/icons-material/Check';
+import { IconButton } from '@mui/material';
+import classNames from 'classnames';
+import globalize from 'scripts/globalize';
+import { useTogglePlayedMutation } from 'hooks/useFetchItems';
+
+interface PlayedButtonProps {
+    className?: string;
+    playedState : boolean | undefined;
```

Comment:
I think I would flip these... `isPlayed` for the prop and `playedState` for the state. :man_shrugging: